<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolWindow</class>
 <widget class="QFrame" name="ToolWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton {
	color:white;
	font: 18pt &quot;微软雅黑&quot;;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolButton" name="toolButtonBack">
     <property name="text">
      <string>Back</string>
     </property>
     <property name="icon">
      <iconset resource="toolwindow.qrc">
       <normaloff>:/tool_icons/Back.png</normaloff>:/tool_icons/Back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonAssistant">
     <property name="text">
      <string>Assistant</string>
     </property>
     <property name="icon">
      <iconset resource="toolwindow.qrc">
       <normaloff>:/tool_icons/Assistant.png</normaloff>:/tool_icons/Assistant.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonHelp">
     <property name="text">
      <string>Help</string>
     </property>
     <property name="icon">
      <iconset resource="toolwindow.qrc">
       <normaloff>:/tool_icons/Help.png</normaloff>:/tool_icons/Help.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonRestart">
     <property name="text">
      <string>Restart</string>
     </property>
     <property name="icon">
      <iconset resource="toolwindow.qrc">
       <normaloff>:/tool_icons/Restart.png</normaloff>:/tool_icons/Restart.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonShutdown">
     <property name="text">
      <string>Shutdown</string>
     </property>
     <property name="icon">
      <iconset resource="toolwindow.qrc">
       <normaloff>:/tool_icons/Shut Down.png</normaloff>:/tool_icons/Shut Down.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="toolwindow.qrc"/>
 </resources>
 <connections/>
</ui>
